build.sbt

```diff
@@ -102,15 +102,20 @@ lazy val semanticdbScalacCore = project.in(file("semanticdb/scalac/library")).se
   sharedSettings,
   publishJVMSettings,
   fullCrossVersionSettings,
-  protobufSettings,
   mimaPreviousArtifacts := Set.empty,
   moduleName := "semanticdb-scalac-core",
   description := "Library to generate SemanticDB from Scalac 2.x internal data structures",
-  Compile / unmanagedSourceDirectories ++= {
-    val base = (ThisBuild / baseDirectory).value / "semanticdb"
-    List(base / "metap", base / "cli", base / "semanticdb", base / "metacp", base / "symtab")
-  },
   libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
+).dependsOn(semanticdbShared)
+
+lazy val semanticdbShared = project.in(file("semanticdb/semanticdb")).settings(
+  moduleName := "semanticdb-shared",
+  sharedSettings,
+  publishJVMSettings,
+  fullCrossVersionSettings,
+  protobufSettings,
+  mimaPreviousArtifacts := Set.empty,
```

Comment:
does this mean semanticdb stuff is not subject to mima checks?

Reply:
That would probably make it so, alternative is probably to just disable mima plugin on it

Reply:
I'm actually curious why disabling is desired...

Reply:
Maybe it is/was a MiMa limitation due to the fact that these projects are fully cross-built?

Reply:
i am not sure. but it's also true that if someone does use the library (to read/parse generated semanticdb files), it's basically protobuf anyway, and they are backwards-compatible.